src/storage.cc

```diff
@@ -10,7 +10,7 @@ Storage::Storage(cyclus::Context* ctx)
       latitude(0.0),
       longitude(0.0),
       coordinates(latitude, longitude) {
-  inventory_tracker.Init({&inventory, &stocks, &ready, &processing}, 1e299);
+  inventory_tracker.Init({&inventory, &stocks, &ready, &processing}, cyclus::CY_LARGE_DOUBLE);
```

Comment:
We can probably drop this argument from the call to `Init()`.  The default value for this argument is already defined as our max int in the header in cyclus/cyclus.  @nuclearkatie - thoughts?

Reply:
Let's keep this the way it is.  This value is specified in the header as the default.